deploy/helm-chart/kubernetes-replicator/templates/deployment.yaml

```diff
@@ -3,40 +3,44 @@ kind: Deployment
 metadata:
   name: {{ include "kubernetes-replicator.fullname" . }}
   labels:
-    app.kubernetes.io/name: {{ include "kubernetes-replicator.name" . }}
-    helm.sh/chart: {{ include "kubernetes-replicator.chart" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    {{- include "kubernetes-replicator.labels" . | nindent 4 }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
```

Comment:
This was previously hardcoded to `1` on purpose -- there's no locking/synchronization mechanism between instances of this controller, so running more than 1 of them would be quite racy.